src/components/Disputes/Import.js

```diff
@@ -68,45 +70,62 @@ function Import() {
   async function handleProjectCreationn() {
     let nameInput = inputName.current.value
     let budgetInput = inputBudget.current.value
+    let dropdownInput = inputDropdown.current.value
     console.log(nameInput, budgetInput)
-    await create('123', nameInput, budgetInput)
+    await create(dropdownInput, nameInput, budgetInput)
   }
 
   return (
     <>
-      {wallet.account !== null ? (
+      {wallet.account !== null &&
+      localStorage.getItem('ACCESS TOKEN') !== null ? (
         <Button mode='strong' onClick={listUserRepos}>
           Import Project
         </Button>
       ) : null}
 
       <SidePanel onClose={handleClose} title='Repository' opened={opened}>
-        <DropDown
-          selected={selected}
-          onChange={handleOnChange}
-          items={[...list]}
-          style={{ marginTop: '4rem' }}
-        />
-
-        <TextInput
-          ref={inputName}
-          style={{ marginTop: '2rem' }}
-          placeholder='Provide a Name'
-        />
-
-        <TextInput
-          ref={inputBudget}
-          style={{ marginTop: '2rem' }}
-          placeholder='Budget'
-        />
-
-        <Button
-          onClick={handleProjectCreationn}
-          style={{ marginTop: '4rem' }}
-          mode='positive'
-        >
-          Import Project
-        </Button>
+        {list.length === 0 ? (
+          <div
+            style={{
+              display: 'flex',
+              justifyContent: 'center',
+              marginTop: '4rem',
+            }}
+          >
+            <LoadingRing mode='half-circle' />
+          </div>
+        ) : (
+          <>
+            <DropDown
+              selected={selected}
+              onChange={handleOnChange}
+              items={[...list]}
+              ref={inputDropdown}
```

Comment:
`inputDropdown` ref is not defined.
  
const inputDropdown = useRef(null)

Reply:
Fixed